tests/test_data_nofile.py

```diff
@@ -45,7 +45,14 @@ def test_natural_abund_excited_state():
     nabund = data.natural_abund(excited)
     assert_equal(nabund, data.natural_abund_map.get(excited))
 
-
+def test_elements():
+    # set the datapath to nonsense so we call, the cpp data version
+    pyne_conf.NUC_DATA_PATH = b'bobbobhonkeytonk'
+    # initialize natural_abund_map
+    # test a series of elements
+    assert_equal(data.atomic_mass('H'),1.0079407540557774)
+    assert_equal(data.atomic_mass('Li'),6.9400366029079965)
+    assert_equal(data.atomic_mass('U'),238.02891048471406)
```

Comment:
Just a note so you aren't surprised, but when asking for the atomic_mass of elements only valid results are returned for naturally occuring elements, for z>92 you get 0.0. This is the same if you use a nuc_data.h5 file.